### DISCUSSION TITLE
Half "empty" screen when using i3 with lxqt

### DISCUSSION THREAD

**Discussion by franzmondlichtmann** [lxqt/lxqt#2551] [Q&A]
I don't know if this is a bug of i3 or lxqt or a bugged out app, so I ask here.
When I use lxqt with i3 and disable the i3-bar, I get a empty space wherever another app would be.
Basically I see the wallpaper in its place, as if there was an invisible application.
I can just move this invisible app like any other application.
(Basically the only relevant change I did was removing the i3 bar section of the i3-config file.)

**Comment by franzmondlichtmann** [comment]
![image](https://github.com/lxqt/lxqt/assets/105480088/b8f5e518-ecb6-4d98-9327-cf01231aefac)
![image](https://github.com/lxqt/lxqt/assets/105480088/e207ea3a-d39d-489d-94b3-7279352b2b1c)
![image](https://github.com/lxqt/lxqt/assets/105480088/84229ea3-4a07-4e31-81f4-f420c3e3d3c5)

**Reply by tsujan** [reply]
You can rule out LXQt as the cause.

**Reply by franzmondlichtmann** [reply]
I found  a clue, the blank window is pcmanfm-qt, after killing this program the blank window disappears.
I may write a bug report there.
